src/main/scala/com/dwolla/aws/lambda/Handler.scala

```diff
@@ -0,0 +1,53 @@
+package com.dwolla.aws.lambda
+
+import aws.AWS.CloudWatch
+import aws.cloudwatch._
+import aws.lambda._
+import cats.effect._
+import com.dwolla.aws.cloudwatch._
+import com.dwolla.aws.lambda.ScheduledEventStream._
+import fs2._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.language.{higherKinds, postfixOps}
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+import scala.scalajs.js.{JSON, _}
+
+object Handler {
+
+  @JSExportTopLevel("removeCloudWatchAlarms")
+  val handler: ScheduledHandler = (event, _, callback: LambdaCallback[Unit]) ⇒ {
+    Stream.emit(event)
+      .covary[IO]
+      .through(handleScheduledEvents[IO])
+      .compile
+      .drain
+      .unsafeRunAsync {
+        case Left(t) ⇒ callback(js.Error(t.getMessage), js.undefined)
+        case Right(()) ⇒ callback(null, ())
```

Comment:
I think the answer is no, but want to confirm anyway.  Do we have to respond to the AutoScalingGroup after receiving this event?

Reply:
No, this event is just informational, not part of a workflow.